frontend/apps/web/src/pages/all-tests/index.tsx

```diff
@@ -0,0 +1,285 @@
+import { PageContainer } from '@/containers/PageContainer'
+import {
+  Button,
+  Colors,
+  Link,
+  PageLayout,
+  ProgressBar,
+  Table,
+  Text,
+  usePageLayout,
+} from '@filecoin/ui'
+import { styled } from '@storybook/theming'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+const Dashboard = () => {
+  const {
+    state: { model },
+  } = PageContainer.useContainer()
+  const allTests = model.getAllTests()
+  const navigate = useNavigate()
+
+  const Header = props => {
+    const navigate = useNavigate()
+    return (
+      <PageLayout.Header>
+        <HeaderWrapper>
+          <Text type="heading 5">All Tests</Text>
+          <Buttons>
+            <Button
+              onClick={() => navigate('/repository-details')}
+              variant="outline"
+              size="medium"
+            >
+              All Behaviours
+            </Button>
+          </Buttons>
+        </HeaderWrapper>
+      </PageLayout.Header>
+    )
+  }
+
+  const prepareAllTestsChart = () => {
+    return Object.entries(
+      allTests.reduce(
+        (d, test) => {
+          if (test.kind === 'e2e') {
+            d.e2e += 1
+          } else if (test.kind === 'benchmark') {
+            d.benchmark += 1
+          } else if (test.kind === 'api') {
+            d.api += 1
+          } else if (test.kind === 'integration') {
+            d.api += 1
+          } else if (test.kind === 'unit') {
+            d.api += 1
+          }
+          return d
+        },
+        {
+          e2e: 0,
+          benchmark: 0,
+          api: 0,
+          integration: 0,
+          unit: 0,
+        },
+      ),
+    ).map(([key, count]) => ({
+      name: key,
+      percentage: (count / allTests.length) * 100,
+    }))
+  }
+
+  const allTestsKinds = prepareAllTestsChart()
+
+  const prepareAllTestsStatus = () => {
+    return Object.entries(
+      allTests.reduce(
+        (d, test) => {
+          if (test.status === 'pass') {
+            d.pass += 1
+          } else if (test.status === 'fail') {
+            d.fail += 1
+          } else {
+            d.missing += 1
+          }
+          return d
+        },
+        {
+          pass: 0,
+          fail: 0,
+          missing: 0,
+        },
+      ),
+    ).map(([key, count]) => ({
+      name: key,
+      percentage: (count / allTests.length) * 100,
+    }))
+  }
+  const allTestsStatus = prepareAllTestsStatus()
+  const pageLayout = usePageLayout({
+    header: <Header />,
+    footer: <PageLayout.Footer />,
+  })
+  return (
+    <PageLayout {...pageLayout}>
+      <PageLayout.Section>
+        <BarWrapper>
+          <Text type="text xl">All Tests Kinds</Text>
+          <ProgressBar
+            legend
+            onClick={() => navigate('/repository-details')}
```

Comment:
We have an issue for this, onClick should be optional, and actually click on Progress will never do anything, at least it shouldn't.

Reply:
Done